README.md

```diff
@@ -38,10 +41,11 @@ There is (or least should be…) a *Type* class for every Collmex record type ("
 Using Composer, just add it to your `composer.json` by running:
 
 ```shell
-composer require mjaschen/collmex
+composer require mjaschen/collmex:^0.6.20
```

Comment:
Maybe you can explain the reason for adding the version number constraint?

Composer adds the current stable version automatically to the `composer.json` when installing the library with `composer require mjaschen/collmex`:

```shell
% composer require mjaschen/collmex
Using version ^0.6.21 for mjaschen/collmex
[...]
% cat composer.json
{
    "require": {
        "mjaschen/collmex": "^0.6.21"
    }
}
```

Am I missing something?

Reply:
Oh, I wasn't aware of this additional behavior. I'll update my PR in a minute.